packages/ts/react-crud/src/autoform.tsx

```diff
@@ -76,8 +76,14 @@ export function ExperimentalAutoForm<TItem>({
           <AutoFormField key={propertyInfo.name} propertyInfo={propertyInfo} form={form} disabled={disabled} />
         ))}
       {formError ? <div style={{ color: 'var(--lumo-error-color)' }}>{formError}</div> : <></>}
-      <HorizontalLayout style={{ marginTop: 'var(--lumo-space-m)' }}>
+      <HorizontalLayout theme="spacing" style={{ marginTop: 'var(--lumo-space-m)', alignSelf: 'flex-end' }}>
+        {form.dirty ? (
```

Comment:
I'm not a UX designer or anything, but I was thinking that the primary button (in this case the submit button) should appear first to make it more accessible for the end user.

Reply:
Actually very typical to put the primary action on the far right side. The CRUD component does the same. But again, a UX expert should decide this 🙂.